Userland/Services/Taskbar/ClockWidget.cpp

```diff
@@ -226,4 +231,22 @@ void ClockWidget::jump_to_current_date()
     m_selected_calendar_button->set_text(m_calendar->formatted_date());
 }
 
+int ClockWidget::time_width()
+{
+    auto width_string = m_time_format_to_width_string.get(m_time_format);
```

Comment:
It seems a bit overkill for these to be stored in a hashmap, can we just return constant sizes from this function based on the input format?